conbench/entities/history.py

```diff
@@ -138,6 +138,14 @@ def get_history_for_benchmark(benchmark_result_id: str):
     # exception.
 
     benchmark_result: BenchmarkResult = BenchmarkResult.one(id=benchmark_result_id)
+
+    if benchmark_result.run.commit is None:
+        # Alternatively, raise an exception here -- allowing to inform the
+        # user that conceptually there will never be history for this
+        # benchmark result, because it's not associated with repo/commit
+        # information
+        return []
```

Comment:
Yeah, maybe it's better to raise an exception here and handle it in the UI. It would be nice to list this as the reason why we can't display a plot, for example.

Reply:
Good. Let's handle this in a follow-up.